fury/ui.py

```diff
@@ -3652,16 +3665,7 @@ def update(self):
         # Populate slots according to the view.
         for i, choice in enumerate(values_to_show):
             slot = self.slots[i]
-            char_width = slot.textblock.size[0] - self.margin
-            permissible_chars = int(self.slot_width)//char_width
-            total_chars = len(str(choice))
-            if total_chars > permissible_chars:
-                excess_chars = total_chars - permissible_chars
-                wrapped_choice = choice[:(-excess_chars) + 3] + "..."
-                slot.element = choice
-                slot.textblock.message = wrapped_choice
-            else:
-                slot.element = choice
```

Comment:
you should keep this whole block

Reply:
Wow, I really don't remember removing any lines from the test_ui.py or ui.py.
but thanks for pointing out I'll look into it.